.ci/Makefile

```diff
@@ -0,0 +1,16 @@
+SHELL = /bin/bash -eo pipefail
+
+.PHONY: release-notes
+release-notes: validate-branch-name
```

Comment:
This task will be enriched with further details in the release notes, for now, it's just as simple as possible.